wordpress-mobile/WordPress-iOS
# [Site creation] Implementation of the Segments Service

Implements #10316 

- The actual implementation of the service is done in WordPressKit-iOS (see list of associated PRs)
- Some model objects are therefore pushed down to WordPressKit
- Updating the reference to `UIDeviceIdentifier` required touching two lines in `AppDelegate` to remove two warnings

See also these two associated PRs:
- [WordPressKit-iOS #71](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/71)
- [WordPressAuthenticator-iOS #46](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/46)

To test:
- Checkout the branch and run the tests
- Navigate to the enhanced site creation flow. Notice the list of Segments, and that the screen after selecting a segment reflects the selection.

Update release notes:
- The implementation is still hidden behind a feature flag and therefore not visible by end users.

cc @kwonye
